portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -14,20 +14,27 @@
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.FetchOptions;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/comments")
 public class CommentsServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    String queryString = request.getQueryString();
+    String[] split = queryString.split("="); 
```

Comment:
Just for the future, as this parsing gets more complex, you might want to look at http://go/mdn/API/URL

Reply:
(I think that class also support URL construction too)

Reply:
Noted thanks for the link :)